core/src/test/java/com/graphhopper/routing/AlternativeRouteEdgeCHTest.java

```diff
@@ -41,8 +37,10 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class AlternativeRouteEdgeCHTest {
-    private final FlagEncoder carFE = FlagEncoders.createCar(new PMap().putObject("turn_costs", true));
-    private final EncodingManager em = EncodingManager.create(carFE);
+    private final BooleanEncodedValue accessEnc = new SimpleBooleanEncodedValue("access", true);
+    private final DecimalEncodedValue speedEnc = new DecimalEncodedValueImpl("speed", 5, 5, false);
+    private final DecimalEncodedValue turnCostEnc = TurnCost.create("car", 1);
+    private final EncodingManager em = EncodingManager.start().add(accessEnc).add(speedEnc).addTurnCostEncodedValue(turnCostEnc).build();
```

Comment:
This is basically the new way to add 'vehicle' encoded values to the EncodingManager. We could do this using the `VehicleEncodedValues`, but I think they should rather be removed.

Reply:
Fixed in #2614